### DISCUSSION TITLE
Can we add search to Mill's documentation site?

### DISCUSSION THREAD

**Discussion by hmf** [com-lihaoyi/mill#1307] [Q&A]
Is it possible to add search to Mill's [antora](https://antora.org/) documentation site? If it is not supported, maybe [this](https://github.com/Mogztter/antora-lunr) will do?

I am asking because I was looking for information on the `dep` interpolator, and it would make such searches easier. 

TIA

**Comment by lefou** [comment]
Yeah, I think that should be doable. The `docs.antora` module maintains a local npm directory, so some patching to enable `antora-lunr` should be possible. Without any experience with lunr, I like the idea of using a local solution which uses a pre-generated index.
